photo_prism_yaml_to_exif.pl

```diff
@@ -33,14 +33,14 @@
       }
     , yaml_dir              => {
           type      => '=s'
-        , env       => '-'
+        , env       => 'YAML_DIR'
         , default   => cwd()
```

Comment:
the env property affects _only_ the help message text, so for env vars to be useful, this line needs changed to:
```perl
, default => $ENV{'YAML_DIR'} || cwd()
```

Reply:
Ah yes. I think I was crediting the Getopt module with too much intelligence to go and get the env vars itself. Fixed now